style.css

```diff
@@ -0,0 +1,157 @@
+@import url("https://fonts.googleapis.com/css2?family=Crete+Round&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Crete+Round&family=Roboto:wght@100&display=swap");
```

Comment:
- [ ] You did a good job importing the font families used in the project, but there are some parts of the project in which you use different font weights, and they must be imported too. Kindly import the font families with all the font weights used in the project so they can work properly.